.github/workflows/e2e-nightly-master.yml

```diff
@@ -41,6 +41,11 @@ jobs:
         working-directory: test/e2e
         run: ./run-multiple.sh networks/nightly/${{ matrix.p2p }}/*-group${{ matrix.group }}-*.toml
 
+      - name: Run reproducer test cases
+        working-directory: test/e2e
+        run: ./build/runner -f networks/apphash-fail.toml
```

Comment:
this should probably be in a separate directory (e.g. `networks/regression` that we can run with some kind of run-multiple.sh glob, so that we don't have to edit this file for every one off.

Also I believe that this suite will run 12 times, because it runs in the matrix build for group times p2p

Reply:
Moved to separate github workflow.